client/src/views/client-area/PersonalDataPage.vue

```diff
@@ -75,14 +86,91 @@ function createItem(
 }
 
 onMounted(async () => {
-  personalData.value = await BmsAccessInstance.get().getPersonalInformation();
+  await getPersonalData();
 });
 
-function openAuthenticationModal(): void {}
+async function getPersonalData(): Promise<void> {
+  personalData.value = await BmsAccessInstance.get().getPersonalInformation();
+}
 
-function openPersonalInfoModal(): void {}
+async function openAuthenticationModal(): Promise<void> {
+  const modal = await modalController.create({
```

Comment:
Check that personalData is set and exit early if it isn't. `?` should not be used as a substitute for checks xD

Reply:
done